fetch_firmware.sh

```diff
@@ -28,11 +28,11 @@ if [[ $(uname -a) == *NixOS* ]]; then
 fi
 
 BR_VERSION=$(git describe --abbrev=0 --tags)
-FW_VERSION=${1:-v2.1.0-develop-2019011120}
+FW_VERSION=${1:-v2.1.0-develop-2019012817-7-gcc824764}
```

Comment:
I think this can be synced with master now since we merged the required open-amp changes?